From 63cd7388c17e4b1009f86bceadc30f8ff8521843 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Fri, 16 Jun 2023 10:55:45 +0530
Subject: [PATCH] Atrace service is deprecated in Target 8 and API34

Atrace HAL is deprecated and should not be serviced in
Android 14.

Tracked-On: OAM-110592
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/framework_manifest.xml     | 9 ---------
 groups/device-specific/caas_dev/framework_manifest.xml | 9 ---------
 2 files changed, 18 deletions(-)

diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index 3ee7577..0f2db4d 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -47,15 +47,6 @@
         <version>1</version>
         <fqname>IHealth/default</fqname>
     </hal-->
-    <hal format="hidl">
-        <name>android.hardware.atrace</name>
-        <transport>hwbinder</transport>
-        <version>1.0</version>
-        <interface>
-            <name>IAtraceDevice</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <hal format="hidl">
         <name>android.hardware.thermal</name>
         <transport>hwbinder</transport>
diff --git a/groups/device-specific/caas_dev/framework_manifest.xml b/groups/device-specific/caas_dev/framework_manifest.xml
index 0ecda8d..5de84f5 100644
--- a/groups/device-specific/caas_dev/framework_manifest.xml
+++ b/groups/device-specific/caas_dev/framework_manifest.xml
@@ -47,15 +47,6 @@
         <version>1</version>
         <fqname>IHealth/default</fqname>
     </hal>
-    <hal format="hidl">
-        <name>android.hardware.atrace</name>
-        <transport>hwbinder</transport>
-        <version>1.0</version>
-        <interface>
-            <name>IAtraceDevice</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <hal format="hidl">
         <name>android.hardware.thermal</name>
         <transport>hwbinder</transport>
-- 
2.17.1

